### PR TITLE
Feature/docs paths

### PR DESCRIPTION
**Pull request by shaharblanksela**
Added new dynamic path_rule for the following redirects written on #194 .
For future implementation just add the new path string to the list in 'locals'.

### CODE REVIEW

**Comment by mshidlov** [comment]
terraform plan production

File: terraform/main.tf [hunk 1]
```diff
@@ -150,8 +161,21 @@ resource "google_compute_url_map" "urlmap" {
...
+        url_redirect {
+          host_redirect          = "docs.amplication.com"
+          path_redirect          = "/docs/${path_rule.key}"
+          https_redirect         = true
+          redirect_response_code = "MOVED_PERMANENTLY_DEFAULT"
+          strip_query            = true
+        }
+      }
+    }
+  }
```
Comment (yuval-hazaz) [inline terraform/main.tf:178]: please replace with a dynamic redirect

File: terraform/main.tf [hunk 2]
```diff
@@ -168,27 +192,27 @@ resource "google_compute_url_map" "urlmap" {
...
+
+# resource "google_compute_url_map" "https_redirect-target" {
+#   project = var.project_id
+#   name    = "${var.lb_name}-amplication-https-redirect"
+#   default_url_redirect {
+#     https_redirect         = true
+#     redirect_response_code = "MOVED_PERMANENTLY_DEFAULT"
+#     strip_query            = false
+#   }
+# }
```
Comment (yuval-hazaz) [inline terraform/main.tf]: why did we remove all this?

**Review by yuval-hazaz — CHANGES_REQUESTED** [review]
@shaharblanksela @mshidlov please see my comments

File: terraform/main.tf (hunk 2 above)
Comment (shaharblanksela) [reply terraform/main.tf]: it just for testing i put it all back

File: terraform/main.tf [hunk 3]
```diff
@@ -110,9 +110,19 @@ module "lb-http" {
...
 }
 
+locals {
+  paths = toset([
+    "getting-started",
+    "sync-with-github",
+    "deploy",
+    "cli",
+  ])
+}
```
[the remark this answers is not in the document]
Comment (shaharblanksela) [reply terraform/main.tf:120]: you can look on my issue what I've tried to do. #196

**Pull request merged**